index.html

```diff
@@ -243,7 +255,7 @@ <h3 class="subtitle">We are always ready to tell you about our
         </div>
 
         <div class="tab-content container">
-          <div class="tab-pane is-active" id="pane-1" class="level">
+          <div class="tab-pane is-active" id="pane-1" class="level" aria-labelledby="1">
```

Comment:
Btw this element can have `role="tabpanel"`. Applicable for all below

Reply:
Nice, I thought that I was done this, good catch!